projects/step-core/src/lib/services/bookmark.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Inject, Injectable } from '@angular/core';
+import { BehaviorSubject, Observable, of } from 'rxjs';
+import { StorageProxy } from '../modules/basics/shared/storage-proxy';
+import { LOCAL_STORAGE } from '../modules/basics/shared/storage.token';
+import { Bookmark } from '../client/generated/models/Bookmark';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class BookmarkService extends StorageProxy {
+  private bookmarks$ = new BehaviorSubject<Bookmark[] | undefined>(undefined);
+
+  constructor(@Inject(LOCAL_STORAGE) storage: Storage) {
+    super(storage, 'BOOKMARKS');
```

Comment:
In this case I would define "BOOKMARKS" as a const and reuse over the service

Reply:
Done